{"last_activity_date": 1477361040, "owner": {"display_name": "Jim Caton", "reputation": 6, "profile_image": "https://graph.facebook.com/10104675188671246/picture?type=large", "user_id": 5086391, "user_type": "registered", "link": "http://stackoverflow.com/users/5086391/jim-caton"}, "is_answered": false, "answer_count": 1, "creation_date": 1477359568, "score": 0, "title": "Attempt to open h5py file, returns errorno = 17, error message = &#39;file exists&#39;", "tags": ["python", "numpy", "h5py"], "view_count": 11, "question_id": 40230064, "link": "http://stackoverflow.com/questions/40230064/attempt-to-open-h5py-file-returns-errorno-17-error-message-file-exists"}